DraggableNeatBLASTFeature.js

```diff
@@ -0,0 +1,39 @@
+
+const $ = require('jquery');
+
+define( [
+          'dojo/_base/declare',
+          'dojo/_base/array',
+           'NeatHTMLFeatures/View/Track/NeatFeatures',
+           'WebApollo/View/Track/DraggableHTMLFeatures'
+        ],
+    function( declare,
+            array,
+            NeatFeatureTrack,
+            HTMLFeatureTrack
+           ) {
+
+
+var DraggableNeatBLASTFeatures = declare( [NeatFeatureTrack,HTMLFeatureTrack],{
+constructor: function(args)  {  }
```

Comment:
Also, I couldn't get this to build without adding a comma here to fix the syntax.

```suggestion
constructor: function(args)  {  },
```